client/app/scripts/charts/edge.js

```diff
@@ -5,6 +5,30 @@ import classNames from 'classnames';
 import { enterEdge, leaveEdge } from '../actions/app-actions';
 import { encodeIdAttribute, decodeIdAttribute } from '../utils/dom-utils';
 
+function IsStorageComponent(id) {
+  if (id === 'persitent_volume' || id === 'storage_class' || id === 'persistent_volume_claim') {
+    return true;
+  }
+  return false;
+}
+
+// getAdjacencyClass takes id which contains information about edge as a topology
+// of parent and child node.
```

Comment:
Nice. But please also add an example to this comment. e.g. 

`The ids take the form: "nodeA;storage_class---nodeB;persistent_volume_claim"`

Reply:
Cool! I will use this example as it is!

Reply:
Go for it! Is it accurate though? There is some `slice(1, -1)` logic going on too so I just wasn't sure what the edgeIds look like exactly.

Reply:
Yes Its right!

Reply:
@foot I removed the `slice` part and updated the example. Please review!